templates/experiments/share/widget.html

```diff
@@ -1,7 +1,7 @@
 {# The rendered content of this template must be suitable for a Javascript string constant: #}
 {# - escape script tags #}
 <!-- place this in the <head> section -->
-\<script type="module" src="https://unpkg.com/open-chat-studio-widget@0.4.1/dist/open-chat-studio-widget/open-chat-studio-widget.esm.js"\>\<\/script\>
+\<script type="module" src="https://unpkg.com/open-chat-studio-widget@0.4.8/dist/open-chat-studio-widget/open-chat-studio-widget.esm.js"\>\<\/script\>
```

Comment:
@stephherbers we'll need to release a new version and update this - maybe we should split out the widget changes into a separate PR that we can release before these changes?